pandas/tests/arrays/categorical/test_indexing.py

```diff
@@ -145,3 +145,15 @@ def test_mask_with_boolean_raises(index):
 
     with pytest.raises(ValueError, match='NA / NaN'):
         s[idx]
+
+
+class NonCoercaibleCategorical(Categorical):
```

Comment:
is this spelled right?

can you add some comments to this.

Reply:
Ha, nope... I'm going to try with monkeypatch.